app/assets/scripts/a11n/signin.js

```diff
@@ -23,15 +26,68 @@ import { FormikInputText } from '../components/common/forms/input-text';
 import { createProcessToast } from '../components/common/toasts';
 import { useUser } from '../context/user';
 
+const Backdrop = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 1111;
+`;
+
+const Modal = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${glsp(2)};
+  width: 100%;
+  max-width: 24rem;
+  background-color: #fff;
+  padding: ${glsp(1.5, themeVal('layout.gap.medium'))};
+  border-radius: ${themeVal('shape.rounded')};
+  overflow: auto;
+`;
+
+const ModalContent = styled.div`
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  gap: ${glsp(1.5)};
+  overflow: auto;
+`;
+
 const getHostedAuthUiUrl = (page) => {
   const clientId = config.auth.userPoolWebClientId;
   const returnTo = getAppURL().cleanHref;
   return `${config.hostedAuthUi}/${page}?client_id=${clientId}&response_type=token&redirect_uri=${returnTo}`;
 };
 
+const altToast = {
+  update: (msg) => {
+    // eslint-disable-next-line no-console
+    console.info(msg);
+  },
+  success: (msg) => {
+    // eslint-disable-next-line no-console
+    console.info(msg);
+  },
+  error: (msg) => {
+    // eslint-disable-next-line no-console
+    console.error(msg);
+  }
```

Comment:
Mostly for debug?

Reply:
It's just a fallback, since we're using a `ref` to update toast. There could be a situation where the toast isn't created. In those cases, we'll have the appropriate logs.